README.md

```diff
@@ -108,7 +109,8 @@ wildcard_search_query: Query = Query(wildcard_queries=wildcard_query_list)
 # [timestamp, message]
 matched_log_messages: List[Tuple[int, str]] = []
 
-with ClpIrFileReader(Path("example.clp.zst")) as clp_reader:
+# A convenience file reader class is also available to directly interact with IRstream compressed log file
```

Comment:
```suggestion
# A convenience file reader class is also available to interact with a file that
# represents an encoded CLP IR stream directly.
```
80-char limit
Also we never used the term "compressed", so we should stick to the concept of "encoded" CLP IR stream